### PR TITLE
add clear pulsar topic interface.

### PR DESCRIPTION
**Pull request by Lico-Tom**

### CODE REVIEW

File: src/app/pulsar/pulsar-instance/pulsar-instance.component.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.example-button-row button,
```
Comment (Shoothzj) [inline src/app/pulsar/pulsar-instance/pulsar-instance.component.scss:1]: does this scss really needed?

File: src/service/pulsar/pulsar.instance.service.ts [hunk 2]
```diff
@@ -15,4 +15,16 @@ export class PulsarInstanceService {
     return this.http.get<PulsarInstance[]>(BACKEND_HOST + "/api/pulsar/instance")
   }
 
+  clean_topic_no_subscription(type: string): Observable<void> {
+    return this.http.delete<void>(BACKEND_HOST + "/api/pulsar/instances/instance/clear-inactive-topic?type" + type)
+  }
+
+  importPulsarFile(): Observable<string> {
```
Comment (Shoothzj) [inline src/service/pulsar/pulsar.instance.service.ts:22]: camel or lower case _

File: src/service/pulsar/pulsar.instance.service.ts [hunk 3]
```diff
@@ -15,4 +15,16 @@ export class PulsarInstanceService {
     return this.http.get<PulsarInstance[]>(BACKEND_HOST + "/api/pulsar/instance")
   }
 
+  clean_topic_no_subscription(type: string): Observable<void> {
+    return this.http.delete<void>(BACKEND_HOST + "/api/pulsar/instances/instance/clear-inactive-topic?type" + type)
```
Comment (Shoothzj) [inline src/service/pulsar/pulsar.instance.service.ts:19]: use post method

File: src/app/pulsar/pulsar-instance/pulsar-instance.component.scss (hunk 1 above)
Comment (Lico-Tom) [reply src/app/pulsar/pulsar-instance/pulsar-instance.component.scss:1]: does't needed. I move it.

File: src/service/pulsar/pulsar.instance.service.ts (hunk 2 above)
Comment (Lico-Tom) [reply src/service/pulsar/pulsar.instance.service.ts]: OK

File: src/service/pulsar/pulsar.instance.service.ts (hunk 3 above)
Comment (Lico-Tom) [reply src/service/pulsar/pulsar.instance.service.ts:19]: OK

**Review by Shoothzj — APPROVED** [review]

**Pull request merged**